lib/turn.rb

```diff
@@ -9,7 +9,8 @@ def initialize(player, board)
   end
 
   def column_select (user_input)
-    @user_selection = [@column_conversion[user_input], player]
+    @user_selection = [@column_conversion[user_input], @player]
+    @board.columns[@user_selection[0]].play_piece(@user_selection[1])
```

Comment:
I will fix the tests on Tuesday. Thank you for the explanation of what this does.